src/api/db/migrate/20170111114943_add_exclude_projects_to_configuration.rb

```diff
@@ -0,0 +1,17 @@
+class AddExcludeProjectsToConfiguration < ActiveRecord::Migration[5.0]
+  def self.up
+    add_column :configurations, :excluded_projects, :string
+    add_column :configurations, :excluded_projects_description, :string
+
+    # update database from file
+    old = CONFIG['global_write_through']
+    CONFIG['global_write_through'] = false
+    ::Configuration.first.update_from_options_yml
+    CONFIG['global_write_through'] = old
```

Comment:
CONFIG['global_write_through'] controls the write (or not) to the backend, so, we disable it before updating the config for only writing in the DB and then we let it as it was before